geode-for-redis/src/main/java/org/apache/geode/redis/internal/commands/RedisCommandType.java

```diff
@@ -477,4 +568,10 @@ public RedisResponse executeCommand(Command command,
 
     return commandExecutor.executeCommand(command, executionHandlerContext);
   }
+
+  public static List<RedisCommandType> getCommandsForCategory(Category type) {
+    return Arrays.stream(RedisCommandType.values())
+        .filter(c -> c.category == type && c.category != Category.IGNORED)
```

Comment:
why do you need "&& c.category != IGNORED"? If it is "== type" then it would not be IGNORED since "type" would never be IGNORED.

Reply:
I had some other use-case in mind. I've replaced `IGNORED` with `UNCATEGORIZED` and removed this check.